docs/Usage/GettingStarted.md

```diff
@@ -34,6 +34,13 @@ When you place the cursor inside a cell and hit **"Run Cell"**, Hydrogen will ex
 ## "Hydrogen: Run All" and "Hydrogen: Run All Above"
 These commands will run all code inside the editor or all code above the cursor.
 
+## "Hydrogen: Toggle Output Area"
+An external output area can be used to display output instead of the inline result view. 
+The output can be displayed either in a scrolling view or a sliding history.
```

Comment:
Want to add a comment about the setting introduced in #1022? 😉

Reply:
Added a bigger and a better gif as I noticed there was a part of "Kap" still visible.